src/components/browse/components/FacetList/FacetTermsList.js

```diff
@@ -9,6 +9,7 @@ import ReactTooltip from 'react-tooltip';
 import { Collapse } from './../../../ui/Collapse';
 import { Fade } from './../../../ui/Fade';
 import { PartialList } from './../../../ui/PartialList';
+import { anyTermsSelected, mergeTerms } from './index';
```

Comment:
Want to avoid importing components from index.js file(s), or from components/files that itself import component that are exporting.

Because then there's some circular dependencies are created - index imports FacetTermsList, FacetTermsList imports index, and things like tree shaking (removing unused/unimported functions from final compiled build.js file) might not work properly.

In most cases an error would be thrown when try to do that, and guessing is working now since mergeTerms isn't called until after constructor and such. Idk. 

----

Would be better to put these into their own file, e.g. `utility-functions.js` or something (and maybe move some other funcs to there as well) _or_ maybe to have them in FacetTermsList, and then in index.js, `import { FacetTermsList, anyTermsSelected, mergeTerms } from './FacetTermsList';`

Reply:
Ok, makes sense.